WifiManager/WifiManager.cpp

```diff
@@ -182,7 +182,16 @@ namespace WPEFramework
 
         uint32_t WifiManager::connect(const JsonObject &parameters, JsonObject &response)
         {
-            LOGINFOMETHOD();
+            if (params.HasLabel("passphrase"))
```

Comment:
Did u mean parameters here instead of params? We need to pass the actual passphrase to the underlying layer and just suppress the logging. Pls verify that it builds and works.

Reply:
Forgot to add one line.
Added commit to fix this.